spec/api-notification-dbus-spec.ts

```diff
@@ -1,126 +0,0 @@
-// For these tests we use a fake DBus daemon to verify libnotify interaction
```

Comment:
Why was this linux test file deleted?

Reply:
Because it used fake daemon which was not designed to appearing of persistent notification support.
I do not agree with next for testing
```// For these tests we use a fake DBus daemon to verify libnotify interaction
// with the session bus. This requires python-dbusmock to be installed and
// running at $DBUS_SESSION_BUS_ADDRESS.
//
// script/spec-runner.js spawns dbusmock, which sets DBUS_SESSION_BUS_ADDRESS.
//
// See https://pypi.python.org/pypi/python-dbusmock to read about dbusmock.```

We have internal core dispatcher and should avoid usage of other dispatchers